app/src/main/java/com/xmartlabs/scasas/geoquiz/ui/QuizActivity.java

```diff
@@ -20,29 +24,9 @@
   private Button falseButton;
   private Button trueButon;
   private Button nextButton;
+  private Button prevButton;
   private TextView questionTextView;
-  private Question[] questionBank = new Question[] {
-      new Question.Builder()
-          .textResId(R.string.question_oceans)
-          .answerTrue(true)
-          .build(),
-      new Question.Builder()
-          .textResId(R.string.question_mideast)
-          .answerTrue(false)
-          .build(),
-      new Question.Builder()
-          .textResId(R.string.question_africa)
-          .answerTrue(false)
-          .build(),
-      new Question.Builder()
-          .textResId(R.string.question_americas)
-          .answerTrue(true)
-          .build(),
-      new Question.Builder()
-          .textResId(R.string.question_asia)
-          .answerTrue(true)
-          .build()
-  };
+  private List<Question> questionBank = new ArrayList<>();
```

Comment:
avoid putting `Bank` or something like that. Call it as `questions`. Also in some case you can use `questionList`